docs/WritingFirstTest-README.md

```diff
@@ -0,0 +1,97 @@
+# Writing the first tests
+
+## Pre-requisites:
+
+        1. Have admin rights  
+        2. Intellij (install cucumber JVM plugin)
+        3. Java JDK 11  
+        4. Gradle 7.1
+        5. Appium
+        6. Android studio with all paths set
+        7. GIT
+        8. Docker 
+        9. Create a fork and Clone the repository from - https://github.com/znsio/teswiz.
```

Comment:
We actually never want people to fork / clone teswiz. They should use getting-started-with-teswiz